build/Makefile.libevent

```diff
@@ -12,7 +12,7 @@ all: Makefile
 	$(MAKE) install
 
 Makefile: configure
-	./configure --prefix=$(INSTALL_DIR) --disable-shared --enable-static --enable-openssl CPPFLAGS="-I$(INSTALL_DIR)/include" CFLAGS="-fPIC" LDFLAGS="-fPIC -L$(INSTALL_DIR)/lib $(EXTRA_LDFLAGS)" || tail -1000 config.out
+	./configure --prefix=$(INSTALL_DIR) --disable-shared --enable-static --enable-openssl CPPFLAGS="-I$(INSTALL_DIR)/include" CFLAGS="-fPIC" LDFLAGS="-fPIC -L$(INSTALL_DIR)/lib" OPENSSL_LIBADD="$(EXTRA_LDFLAGS)" || tail -1000 config.out
```

Comment:
No, I did one better: I ran it on `certificate-transparency`, not `openssl`. :smiley: 

I thought I was reading `configure-ct`, for some reason, let me get back to you.